src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -350,6 +361,9 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
             zoomControl.addTo(map);
             initializeOverlays();
             initializeLayerControl.apply(this, null);
+            if (currentLocationMarker) {
+                currentLocationMarker.addTo(map);
+            }
```

Comment:
Do you get geolocation errors in the console when using the home page `from` geolocator button? It may be the location simply wasn't found.